plinth/templates/base.html

```diff
@@ -222,5 +222,6 @@
 
   {% block app_js %}<!-- placeholder for app-specific js files -->{% endblock %}
   {% block page_js %}<!-- placeholder for page-specific js files -->{% endblock %}
+  <a href="{% static 'jslicense.html' %}" data-jslicense="1">JavaScript license information</a>
```

Comment:
I suggest to move this inside the footer_block, and center the text. That should improve the appearance a bit.